apps/storefront/.env.template

```diff
@@ -3,8 +3,17 @@ SCOS_BASE_URL=
 
 # The STORE env variable is optional and can be used to initialise the application to use a 
 # specific store. When this variable is not provided, the first store from the backend would be used. 
-# STORE=DE
+STORE=DE
```

Comment:
Why did we make DE hardcoded again? 
- if we do not specify a STORE we take the 1st one, so we should be good
- a hardcoded store will block us from using multi-store

Reply:
this file is just a template, it is not used anywhere

---

I've uncommented it because all other params are not commented (to make it consistent as well)